gustaf/create/faces.py

```diff
@@ -33,3 +34,129 @@ def box(bounds=[[0, 0], [1, 1]], resolutions=[2, 2]):
     face_mesh = Faces(vertex_mesh.vertices, connectivity)
 
     return face_mesh
+
+def simplexify(quad, backslash=False, alternate=True):
```

Comment:
```suggestion
def tosimplex(quad, backslash=False):
```
There's no guarantee in alternate and I don't think it should be an option. Maybe you can put a separate example with this